core/embed/extmod/modtrezorio/modtrezorio-usb.h

```diff
@@ -190,7 +190,9 @@ STATIC mp_obj_t mod_trezorio_USB_open(mp_obj_t self,
   mp_obj_get_array(MP_OBJ_FROM_PTR(&o->ifaces), &iface_cnt, &iface_objs);
 
   // Initialize the USB stack
-  usb_init(&o->info);
+  if (sectrue != usb_init(&o->info)) {
+    mp_raise_ValueError("failed to initialize usb driver");
```

Comment:
`ValueError` doesn't fit well i think, how about
`mp_raise_msg(&mp_type_RuntimeError, "failed to initialize usb driver");`

Reply:
fixed, see f0032e5a3818a4cfffe7e5fd200c572106d882cc